src/filefinder_rtp.cpp

```diff
@@ -131,6 +131,11 @@ FileFinder_RTP::FileFinder_RTP(bool no_rtp, bool no_rtp_warnings) {
 		env_paths.insert(env_paths.end(), tmp.begin(), tmp.end());
 	}
 
+	// If a custom RTP path is set, use this with highest precedence
+	if (!rtp_path.empty()) {
+		env_paths.insert(env_paths.begin(), rtp_path);
```

Comment:
imo this should support Tokenizing (multiple paths split by ; or :) like the other env_paths. See the other cases a few lines above.

Reply:
Added tokenizing support.